src/builders/payload_builder.ts

```diff
@@ -371,19 +370,20 @@ Kwil Chain ID: ${tx.body.chain_id}
     return Txn.copy<BytesEncodingStatus.BASE64_ENCODED>(tx, (newTx) => {
       newTx.signature = {
         // bytes must be base64 encoded for transport over GRPC
-        signature_bytes: bytesToBase64(signedMessage),
-        signature_type: signatureType.toString() as SignatureType,
+        sig: bytesToBase64(signedMessage),
+        type: signatureType.toString() as SignatureType,
       };
       newTx.body = {
-        description: description,
+        desc: description,
         payload: bytesToBase64(tx.body.payload as Uint8Array),
-        payload_type: newTx.body.payload_type as PayloadType,
-        fee: newTx.body.fee?.toString(),
+        type: newTx.body.type as PayloadType,
+        // @ts-ignore - DO NOT MERGE WITHOUT RESOLVING - Jon is looking into why fee is unmarshalled to an integer.
+        fee: Number(newTx.body.fee),
```

Comment:
Pretty sure the resolution is simple, but need to make sure this is as expected: https://github.com/kwilteam/kwil-db/pull/713